src/snowflake/cli/plugins/snowpark/package_utils.py

```diff
@@ -38,11 +37,13 @@ def parse_requirements(
     """
     reqs: List[Requirement] = []
     if requirements_file.exists():
-        with requirements_file.open(
-            "r", read_file_limit_mb=DEFAULT_SIZE_LIMIT_MB, encoding="utf-8"
-        ) as f:
-            for req in requirements.parse(f):
-                reqs.append(req)
+        for line in requirements_file.read_text(
+            file_size_limit_mb=DEFAULT_SIZE_LIMIT_MB
+        ).splitlines():
```

Comment:
Isn't there `readlines` method?

Reply:
Not in pathlib - I'd have to convert that back to "with file.open(...)"